executor/seqtest/seq_executor_test.go

```diff
@@ -760,43 +760,6 @@ func (s *seqTestSuite) TestAdminShowNextID(c *C) {
 	r.Check(testkit.Rows("test1 tt id 41"))
 }
 
-func (s *seqTestSuite) TestNoHistoryWhenDisableRetry(c *C) {
```

Comment:
removed?

Reply:
This feature requires history to be always saved. As mentioned in the PR description, #11192 is basically reverted.